src/modules/uniplayer/Uniplayer.jsx

```diff
@@ -116,31 +116,46 @@ export default class Uniplayer extends React.Component {
  renderYT(){
    var player = {};
    player.id = 'https://www.youtube.com/watch?v=' + '2H5R0bdblEE';
+   player.background = 'http://i3.ytimg.com/vi/' + '2H5R0bdblEE' + '/maxresdefault.jpg';
```

Comment:
We will be able to host our own video via cloudfront CDN which will be faster and eliminate CORS issues. At the moment only I have permissions to do that but when I create AWS users for you guys I will give the users permission to add things to our public bucket.

Reply:
We should probably just remove the image for videos (youtube) and only have the render for music (soundcloud).